src/main/java/org/spongepowered/common/service/permission/SpongePermissionService.java

```diff
@@ -180,4 +185,13 @@ public void addDescription(PermissionDescription permissionDescription) {
     public SpongeSubjectCollection getDefaultCollection() {
         return defaultCollection;
     }
+
+    public void setupDefaultPermissions() {
```

Comment:
Oops, I thought I merged your forge commit as well into my local bleeding but apparently forgot to do so.